SalesforceCommerceCloud/pwa-kit
# @W-17178104@ - Fixed "getCheckboxProps is not a function" when rendering checkout page in generated app.

# Description



As a developer, I'd like to be able render checkout when I have an account and basket has not had a shipping address yet.
In the generated app, the checkout page throws an error `getCheckboxProps is not a function` when rendering the checkout page.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Fix incorrect import from chakra-ui radio button.

# How to Test-Drive This PR

- Checkout the branch
- generate a project using local npm. Run `node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir ../generated-projects/radio-test` from monorepo root
- Run the generated project (Run `npm start` in generated project)
- Registere a new account
- Add a new address
- Add an Item to basket
- Go to check out page
- Confirm the page renders correctly.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
